Categories/CoreLocation/CLGeocoder+Promise.swift

```diff
@@ -35,6 +35,15 @@ extension CLGeocoder {
             }
         }
     }
+    
+  public func geocode(addressString: String, region: CLRegion) -> Promise<[CLPlacemark]> {
```

Comment:
Oups ok doing it now
I am using carthage so no need to push anything i will use the commit id until next version

Sent from my iPhone


> On Jun 30, 2015, at 12:08 PM, Max Howell <notifications@github.com> wrote:
> 
> In Categories/CoreLocation/CLGeocoder+Promise.swift:
> 
> > @@ -35,6 +35,15 @@ extension CLGeocoder {
> >              }
> >          }
> >      }
> > +    
> > +  public func geocode(addressString: String, region: CLRegion) -> Promise<[CLPlacemark]> {
> Should be CLRegion? if mirroring the underlying API.
> 
> —
> Reply to this email directly or view it on GitHub.
>